sidebars.js

```diff
@@ -12,59 +11,96 @@
 
 module.exports = {
   docs: [
-    'introduction',
+    "introduction",
     {
-      type: 'category',
-      label: 'Getting started',
+      type: "category",
+      label: "Getting started",
       collapsed: false,
       items: [
-        'community/welcome-to-the-community',
-        'community/highlights',
-        'community/100-days-of-oss',
-        'community/faqs'
+        "community/welcome-to-the-community",
+        "community/highlights",
+        "community/100-days-of-oss",
+        "community/faqs",
       ],
     },
     {
-      type: 'category',
-      label: 'Contributing guide',
+      type: "category",
+      label: "Contributing guide",
       collapsed: false,
       items: [
-        'contributing/introduction-to-contributing',
-        'contributing/code-of-conduct',
-        'contributing/triage-guide',
-        'contributing/set-up-authentication'
+        "contributing/introduction-to-contributing",
+        "contributing/code-of-conduct",
+        "contributing/triage-guide",
+        "contributing/set-up-authentication",
       ],
     },
     {
-      type: 'category',
-      label: 'Technical guide',
+      type: "category",
+      label: "Technical guide",
       collapsed: false,
-      items: [
-        'technical/introduction-to-storybook',
-        'technical/resolve-merge-conflicts',
-      ],
+      items: ["technical/introduction-to-storybook", "technical/resolve-merge-conflicts"],
     },
     {
-      type: 'category',
+      type: "category",
       label: `Maintainer guide`,
       collapsed: true,
       items: [
-        'maintainers/setting-up-a-new-repository',
-        'maintainers/check-engines',
-        'maintainers/conventional-commit',
-        'maintainers/semantic-release',
-      ]
+        "maintainers/setting-up-a-new-repository",
+        "maintainers/check-engines",
+        "maintainers/conventional-commit",
+        "maintainers/semantic-release",
+      ],
     },
     {
-      type: 'category',
-      label: 'Chrome Extension Usage',
+      type: "category",
+      label: "Chrome Extension Usage",
       collapsed: true,
       items: [
-        'chrome-extension/introduction-to-the-chrome-extension',
-        'chrome-extension/using-the-chrome-extension',
+        "chrome-extension/introduction-to-the-chrome-extension",
+        "chrome-extension/using-the-chrome-extension",
+        {
+          type: "category",
+          label: "using the extension",
+          items: [
+            {
+              type: "file",
+              label: "feature 1",
+              path: "chrome-extension/ai-code-explanation.md",
+              type: "markdown",
+            },
```

Comment:
Can't use markdown as type and instead of path, use the id. 
```suggestion
            {
              type: "file",
              label: "feature 1",
              type: "doc",
              id: "ai-code-explanation"
            },
```